Could you double-check your syntax? Running `make docs` locally failed:

<pre>
rendering: file
Traceback (most recent call last):
  File "hacking/module_formatter.py", line 415, in <module>
    main()
  File "hacking/module_formatter.py", line 404, in main
    text = template.render(doc)
  File "/usr/lib/python2.7/site-packages/jinja2/environment.py", line 894, in render
    return self.environment.handle_exception(exc_info, True)
  File "hacking/templates/man.j2", line 20, in top-level template code
    {% for desc in v.description %}@{ desc | jpfunc }@{% endfor %}
  File "hacking/module_formatter.py", line 95, in man_ify
    t = _ITALIC.sub(r'\\fI' + r"\1" + r"\\fR", text)
TypeError: expected string or buffer
make: *** [modulepages] Error 1
</pre>

--- a/library/files/file
+++ b/library/files/file
@@ -39,7 +39,7 @@ description:
 options:
   path:
     description:
-      - defines the file being managed, unless when used with C(state=link), and then sets the destination to create a symbolic link to using I(src). Aliases: I(dest), I(name)
+      - 'defines the file being managed, unless when used with C(state=link), and then sets the destination to create a symbolic link to using I(src). Aliases: I(dest), I(name)'
     required: true
     default: []
     aliases: ['dest', 'name'] 

